packages/admin-ui-docs/src/components/StatusTable/components.tsx

```diff
@@ -300,144 +253,77 @@ const layout = [
 const page = [
   {
     component: 'Tab',
-    status: 'supported',
+    status: 'stable',
     notes: '-',
     type: 'page',
   },
   {
     component: 'Stats',
-    status: 'upcoming',
+    status: 'proposed',
     notes: 'Some teams currently refer to this component as Totalizer',
     type: 'page',
   },
-  {
-    component: 'Sidebar',
-    status: 'experimental',
-    notes: 'Will be renamed to Left Nav',
-    type: 'page',
-  },
-  {
-    component: 'Topbar',
-    status: 'experimental',
-    notes: '-',
-    type: 'page',
-  },
-  {
-    component: 'PageHeader',
-    status: 'supported',
-    notes: '-',
-    type: 'page',
-  },
   {
     component: 'Page',
-    status: 'supported',
-    notes: '-',
-    type: 'page',
-  },
-  {
-    component: 'PageContent',
-    status: 'supported',
-    notes: '-',
-    type: 'page',
-  },
-  {
-    component: 'PageTitle',
-    status: 'supported',
-    notes: '-',
-    type: 'page',
-  },
-  {
-    component: 'PageActions',
-    status: 'supported',
+    status: 'stable',
```

Comment:
We'll have changes in the API of this component

```suggestion
    status: 'experimental',
```